test-utils doesn't export /api

```
» npm test --workspace=@skilljar/web-components                                                                           1 ↵

> @skilljar/web-components@0.0.1 test
> jest

 FAIL  tst/icon.test.js
  ● Test suite failed to run

    Cannot find module './api/mount' from '../../node_modules/@tybalt/test-utils/dist/index.js'

    Require stack:
      /Users/dougwade/code/course-platform/node_modules/@tybalt/test-utils/dist/index.js
      tst/icon.test.js

      at Resolver.resolveModule (../../node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (../../node_modules/@tybalt/test-utils/dist/index.js:31:28)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.782 s
Ran all test suites.
npm ERR! Lifecycle script `test` failed with error:
npm ERR! Error: command failed
npm ERR!   in workspace: @skilljar/web-components@0.0.1
npm ERR!   at location: /Users/dougwade/code/course-platform/packages/web-components
```

--- a/packages/test-utils/package.json
+++ b/packages/test-utils/package.json
@@ -28,7 +28,7 @@
     "typescript": "^4.9.4"
   },
   "files": [
-    "dist/*",
+    "dist/**/*",
     "package.json",
     "README.md"
   ]

